package.json

```diff
@@ -120,5 +120,6 @@
       "packages/**"
     ]
   },
-  "dependencies": {}
+  "dependencies": {},
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
This is from the new yarn upgrades. Do we know if this is something that's going to change every build or every version?

Reply:
That sounds like a good question for @jonkoops

Reply:
When [Corepack](https://nodejs.org/api/corepack.html) is enabled it will automatically add this field when running `yarn install` to ensure the package manager remains consistent. If you want to work around this use `corepack disable` when working on the `v6` branch, and then `corepack enable` when working on `main`.

Of course, as soon as `v6` is rebased with `main` this will no longer be an issue and Corepack can remain enabled at all times.